packages/env-utils/src/envUtils.ts

```diff
@@ -91,7 +91,7 @@ const isLinux = () => {
     return getPlatform().startsWith('Linux');
 };
 
-const isCodesignBuild = () => !!process.env.CODESIGN_BUILD;
+const isCodesignBuild = () => process.env.IS_CODESIGN_BUILD === 'true';
```

Comment:
This is now the only place where webpack's defineplugin replace the value of `process.env.IS_CODESIGN_BUILD === 'true';` by `"true"` or `"false"`. Maybe we can think if we need it as a build time variable or runtime would be ok.